src/polybench/POLYBENCH_ATAX.cpp

```diff
@@ -21,7 +21,7 @@ namespace polybench
 POLYBENCH_ATAX::POLYBENCH_ATAX(const RunParams& params)
   : KernelBase(rajaperf::Polybench_ATAX, params)
 {
-  Index_type N_default = 2100;
+  Index_type N_default = 1000;
```

Comment:
Does this make this kernel run fast enogh we should increase the reps?

Reply:
The base seq variant runs in 0.5 - 0.9 secs, depending on the compiler and system. That's reasonable, so I didn't up the reps.